moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -104,9 +104,11 @@ class Servo
   std::string getStatusMessage() const;
 
   /**
-   * \brief Returns the end effector pose in planning frame
+   * \brief Returns the current pose of a specified target frame.
+   * @param target_frame The name of the target frame.
+   * @return The global transform of the specified frame.
    */
-  Eigen::Isometry3d getEndEffectorPose() const;
+  Eigen::Isometry3d getCurrentPose(const std::string& target_frame) const;
```

Comment:
Given that the library is not using this, I would remove it from the API and move into the tests where this is needed.

Reply:
Good call -- done!